README.md

```diff
@@ -1,2 +1,64 @@
 # ibc-apps
-IBC applications and middleware for Cosmos SDK chains
+
+![IBC-APPS Header](ibc-apps.png)
+
+IBC applications and middleware for Cosmos SDK blockchains
+
+🌌 Why have an ibc-apps repo?
+================================
+
+Early IBC work started in the [ibc-go](https://github.com/cosmos/ibc-go) repo. As the repo grew, the need arose to parallelize the work among many teams.  
+
+The ibc-apps repo is meant to be an easily discoverable, navigable, central place for modules and middleware.
+
+🌌🌌 Who's it for?
+===================
+
+IBC-Apps is for: 
+- _Core **ibc-go** contributors_; it frees them from having to maintain IBC Apps,
+
+- _Publishers of **ibc apps**_, so their apps can be easily found, and 
+
+- _Everyone who uses IBC_ and wants to benefit from the full range of its capabilities.
+
+
+🌌🌌🌌 What is it?
+==================
+
+### What is IBC?
```

Comment:
Should we explain this here? Feel like one sentence about IBC and one about ibc-go would be enough with references for further prerequisite readings if the reader needs it. Given the comment above wrt coherency between ibc-go and ibc-apps, how about we use this:

> 
> The [Inter-Blockchain Communication protocol (IBC)](https://ibcprotocol.dev/) allows blockchains to talk to each other. This end-to-end, connection-oriented, stateful protocol provides reliable, ordered, and authenticated communication between heterogeneous blockchains.
> 
> This IBC implementation in Golang is built as a Cosmos SDK module. To understand more about how to use the ibc-go module as well as about the IBC protocol, please check out the Interchain Developer Academy [section on IBC](https://tutorials.cosmos.network/academy/3-ibc/), or [our docs](https://ibc.cosmos.network/main/ibc/overview.html)."

Could be rephrased but likely good to have something similar in both repos